chartfx-math/src/main/java/de/gsi/math/MathGenBase.java

```diff
@@ -2453,187 +2200,70 @@ public static double GamSer(double a, double x) {
      * @param norm normalisation factor
      * @return the computed result
      */
-    public static double Gauss(double x, double mean, double sigma, boolean norm) {
+    public static double gauss(double x, double mean, double sigma, boolean norm) {
         if (sigma == 0) {
             return 1.e30;
         }
         final double arg = (x - mean) / sigma;
-        final double res = Exp(-0.5 * arg * arg);
+        final double res = exp(-0.5 * arg * arg);
         if (!norm) {
             return res;
         }
         return res / (2.50662827463100024 * sigma); // sqrt(2*Pi)=2.50662827463100024
     }
 
+    // codegen: double -> float, int, long, short
     /**
      * geometric_mean = (\Prod_{i=0}^{n-1} \abs{a[i]})^{1/n}
      * 
      * @param a input vector
      * @param length &lt;= data.length elements to be used
      * @return geometric mean of an array a with length n.
      */
-    public static double GeometricMean(double[] a, int length) {
+    public static double geometricMean(double[] a, int length) {
         if (a == null || a.length <= 0) {
             return -1;
         }
-        double logsum = 0.0;
+        double logsum = 0.0; //// codegen: skip all
 
         // use logarithm represenation:
         // product in linear regime -> sum in logarithmic scale
         for (int i = 0; i < length; i++) {
             if (a[i] == 0) {
                 return 0.;
             }
-            final double absa = Math.abs(a[i]);
-            logsum += Math.log(absa);
+            final double absa = java.lang.Math.abs(a[i]); //// codegen: skip all
+            logsum += java.lang.Math.log(absa);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.log' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)